docker/conf/nginx.conf

```diff
@@ -0,0 +1,74 @@
+user nginx;
+worker_processes auto;
+pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+  worker_connections 768;
+}
+
+http {
+  sendfile on;
+  tcp_nopush on;
+  tcp_nodelay on;
+  keepalive_timeout 65;
+  server_tokens off;
+  types_hash_max_size 2048;
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+  ssl_protocols TLSv1.2; # Dropping SSLv3, ref: POODLE
+  ssl_prefer_server_ciphers on;
+  error_log               /dev/stdout info;
+  access_log              /dev/stdout combined;
+  gzip on;
+  include                 /etc/nginx/conf.d/*.conf;
+  include                 /etc/nginx/sites-enabled/*;
+  
+  log_format json_combined escape=json '{'
+      '"@timestamp": "$time_iso8601", '
+      '"message": "$remote_addr - $remote_user [$time_local] \\\"$request\\\" $status $body_bytes_sent \\\"$http_referer\\\" \\\"$http_user_agent\\\"", '
+      '"tags": ["nginx_access"], '
+      '"realip": ""$remote_addr", '
```

Comment:
There's a typo here - `""$remote_addr"` is invalid JSON, it should be `"$remote_addr"`

I'd be interested in knowing where you copied this from, I thought I'd fixed all of these typos...

Reply:
Resolved in [7aadc3f](https://github.com/ministryofjustice/bichard7-next-ui/pull/5/commits/7aadc3f357b1ee4fe2524a56699df31d3117abed)